src/lib.rs

```diff
@@ -193,9 +215,7 @@ impl Kaboodle {
     /// should be stable against ordering differences across different hosts.
     pub async fn fingerprint(&self) -> String {
```

Comment:
Calling out that the public-facing fingerprint is still the full SHA-256 hash, not an 8 character substring.